Cargo.toml

```diff
@@ -11,12 +11,11 @@ repository = "https://github.com/Blissfully/pg_parcel"
 version = "0.4.3"
 
 [dependencies]
-clap = {version = "3.1.5", features = ["derive", "wrap_help"]}
-dialoguer = "0.10.0"
+clap = { version = "3.1.5", features = ["derive", "wrap_help"] }
 indicatif = "0.17.0"
 itertools = "0.10.3"
 lazy_static = "1.4.0"
-postgres = {version = "0.19.4", features = ["array-impls"]}
```

Comment:
Fwiw, I wasn't thinking re. whitespace. These changes seem to be what the TOML mode in VSCode is doing. I'm using [Even Better TOML](https://marketplace.visualstudio.com/items?itemName=tamasfe.even-better-toml). Not sure if that's the norm, but it appears to be configurable in this regard. Maybe we can add a workspace recommendation for this plugin plus some workspace settings to avoid formatting tennis.

Reply:
VS Code settings improvements would be welcomed.